### PR TITLE
Update version for the next release (v0.20.1)

### PR DESCRIPTION
**Pull request by bidoubiwa**
Update version to 0.20.1 See body of release for full update

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -22,5 +22,5 @@ If applicable, add screenshots or logs to help explain your problem.
 
 **Environment (please complete the following information):**
 - OS: [e.g. Debian GNU/Linux]
-- Meilisearch version: [e.g. v.0.20.0]
+- Meilisearch version: [e.g. v.0.29.1]
```
Comment (alallema) [inline .github/ISSUE_TEMPLATE/bug_report.md:25]: Do you sure you want to update the `bug_report` template on the same PR?

Comment (bidoubiwa) [reply .github/ISSUE_TEMPLATE/bug_report.md:25]: I came by that mistake and thought I should fixed it. But if you prefer I can open another PR with it

Comment (bidoubiwa) [reply .github/ISSUE_TEMPLATE/bug_report.md:25]: My bad, you are right it is not needed as an update. Its just an example! Though I'm updating it for rust as it seems less out-dated.

Comment (alallema) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Like you prefer just to be sure

**Review by alallema — APPROVED** [review]
LGTM! 🌯

**Comment by bidoubiwa** [comment]
_This message is sent automatically_

Thanks again for contributing to Meilisearch :heart:
If you are participating in [Hacktoberfest](https://blog.meilisearch.com/contribute-hacktoberfest-2022/), and you would like to receive some gift from Meilisearch too, please complete [this form](https://meilisearch.typeform.com/to/idt5qugY).

**Pull request merged**